egs/aishell/ASR/conformer_ctc/pretrained.py

```diff
@@ -46,27 +46,29 @@ def get_parser():
         "--checkpoint",
         type=str,
         required=True,
-        help="Path to the checkpoint. "
-        "The checkpoint is assumed to be saved by "
-        "icefall.checkpoint.save_checkpoint().",
+        help=(
+            "Path to the checkpoint. "
+            "The checkpoint is assumed to be saved by "
+            "icefall.checkpoint.save_checkpoint()."
+        ),
     )
 
     parser.add_argument(
         "--tokens-file",
         type=str,
-        help="Path to tokens.txt" "Used only when method is ctc-decoding",
+        help="Path to tokens.txtUsed only when method is ctc-decoding",
```

Comment:
There should be a space between ".txt" and "Used".
Looks like `black` cannot handle such a case properly.

Reply:
I think this and the other issues were caused due to using the `--experimental-string-processing` feature which is not stable yet. I will run black again without this flag to fix the issues

Reply:
Thanks